### DISCUSSION TITLE
phenix style mtz

### DISCUSSION THREAD

**Discussion by keitaroyam** [project-gemmi/gemmi#130] [About wwPDB depositions]
I asked my colleague to try this site for unmerged cif preparation, and we found that phenix-style mtz file was not supported.
https://project-gemmi.github.io/wasm/mxdepo.html

Could you also support "R-free-flags" for test set, and "I-obs/SIGI-obs" for intensities?

Full list of phenix-style mtz column labels may be found here:
https://github.com/cctbx/cctbx_project/blob/master/iotbx/cif_mtz_data_labels.py
(Not sure this covers everything)

**Comment by wojdyr** [comment]
That's a great news (that your colleague tried it, not that it didn't work). So far I had only one bug report, so probably not many people use it.

It should be fixed now. I just added these labels and updated the web page.

BTW, CCP4 has many more labels than the ones listed in `cif_mtz_data_labels.py`. For example, for the free flags I had `FREE`, `RFREE`, `FREER`, `FreeR_flag`.

**Reply by keitaroyam** [reply]
Thank you! My colleague tried it again, and it worked.

Regarding CCP4 style free flags, it is also confusing that sometimes people use arbitrary flag values other than zero.

**Reply by wojdyr** [reply]
I hope that datasets with non-default free flags won't be converted. The converter assumes value 0 for the free set.
Unless most of reflections have flag == 0 (PHENIX-style) – then 1 is used for the free set.

**Reply by wojdyr** [reply]
I added an option to use other free flag values, but only in the command-line program that does the same job.
The program is normally run as:

    gemmi mtz2cif --depo INPUT1 INPUT2 OUTPUT

Non-default value of the free flag can be chosen by adding: `--nfree=N`.
